src/tests/integration/test-data/foundry/test/FreshInt.t.sol

```diff
@@ -30,4 +33,32 @@ contract FreshIntTest is Test, KEVMCheats {
         int128 val = int128(uint128(kevm.freshUInt(16)));
         assertGt(val, max);
     }
+
+    function manip_symbolic_bytes(bytes memory b) public {
+        uint middle = b.length / 2;
+        b[middle] = hex'aa';
+    }
+
+    function test_symbolic_bytes_function() public {
+        bytes memory fresh_bytes = kevm.freshBytes(5);
+        manip_symbolic_bytes(fresh_bytes);
+        assertEq(hex'aa', fresh_bytes[2]);
+    }
```

Comment:
We can have a more general test here. Generate symbolic bytes of symbolic length, bound the length to be greater than zero and less than something (say, 42), then use the manipulator function, then look up and test for equality. That is the complexity that we need to be able to handle.

Reply:
Consolidated `test_symbolic_bytes_function` and `test_symbolic_bytes_length` so that the symbolic length and passing to a function are tested in conjunction.